peps/pep-0752.rst

```diff
@@ -422,6 +433,145 @@ and ``Y``. If each repository has both packages but one is malicious on ``X``
 and the other is malicious on ``Y`` then the user would be unable to satisfy
 their requirements without encountering a malicious package.
 
+.. _digital-attestations:
+
+Exclusive Reliance on Digital Attestations
+------------------------------------------
+
+The idea [5]_ here would be to solely rely on :pep:`740` attestations to verify
+certain properties of dependencies, such as:
+
+* The package was uploaded by a specific organization or user name
+* The package was uploaded by an owner of a specific domain name
+* The package was uploaded by a user with a specific email address
+
+A general downside to this approach is that implementing this on custom package
+repositories would require a significant amount of work. In the case of PyPI,
+the prerequisite work on `Trusted Publishing`__ and then the
+`PEP 740 implementation`__ itself took the equivalent of a full-time engineer
+one year whose time was paid for by a corporate sponsor. This is unlikely to be
+implemented by enterprise because it's possible to implement reproducible
```

Comment:
"...by enterprise..." reads a little awkward.  Did you mean *an* enterprise (i.e. a corporate organization)?

Reply:
Fixed, thanks!